distributions/otelcol-ebpf-profiler/.goreleaser.yaml

```
@@ -0,0 +1,97 @@
+version: 2
+project_name: opentelemetry-collector-releases
+env:
+  - COSIGN_YES=true
+  - LD_FLAGS=-extldflags=-static
+  - BUILD_FLAGS=-trimpath
+  - GO_TAGS=osusergo,netgo
```

Should we add `CGO_ENABLED=1` here?
```suggestion
  - GO_TAGS=osusergo,netgo
  - CGO_ENABLED=1
```
`CGO_ENABLED=1` is required for the project.